app/templates/gentelella/admin/event/sessions/base_session_table.html

```diff
@@ -196,7 +236,25 @@
         table.on('draw', function () {
             $('[data-toggle="tooltip"]').tooltip();
         });
+        $('#mailModal').on('show.bs.modal', function (event) {
+  var button = $(event.relatedTarget) // Button that triggered the modal
+  var sessionid = button.data('sessionid') // Extract info from data-* attributes
+  // If necessary, you could initiate an AJAX request here (and then do the updating in a callback).
+  // Update the modal's content. We'll use jQuery here, but you could use a data binding library or other methods instead.
+  var emailtype=button.data('typeofmail')
+  var sessiontitle=button.data('sessiontitle')
+  var modal = $(this)
```

Comment:
Add semicolons at end of statements. It is a bad practice not to put semicolon at end of assignment statements